### DISCUSSION TITLE
docker.io/rabbitmq:3-management-alpine container image list cannot be imported into quay.io

### DISCUSSION THREAD

**Discussion by mh21** [rabbitmq/rabbitmq-server#9133] [Other] [answered]
### Describe the bug

Copying the docker.io/rabbitmq:3-management-alpine container image list to quay.io via `skopeo copy --all` worked up to  Jul 31, 2023, when it started to fail with `application/vnd.oci.image.index.v1+json(uploading manifest 3-management-alpine to [quay.io/cki/mirror_rabbitmq](http://quay.io/cki/mirror_rabbitmq): manifest invalid), application/vnd.docker.distribution.manifest.list.v2+json(uploading manifest 3-management-alpine to [quay.io/cki/mirror_rabbitmq](http://quay.io/cki/mirror_rabbitmq): manifest invalid)`.

Starting this date, the image list manifest changed to `application/vnd.oci.image.index.v1+json`. Discussing this with the quay.io folks, they were pointing at one of the referenced manifests having the mediaType of `application/vnd.docker.distribution.manifest.v2+json` as the most likely reason.

With this, it was possible to workaround this via the `--format oci` parameter of skopeo which converts the manifest above to `application/vnd.oci.image.manifest.v1+json` which makes the copy to go through.

### Reproduction steps

1. get a quay.io account
2. skopeo login quay.io
3. on e.g. a recent Fedora container, run
4. skopeo copy --all docker://docker.io/rabbitmq:3-management-alpine docker://quay.io/foo/bar:baz

### Expected behavior

container image list can be copied to quay.io successfully

**Comment by michaelklishin** [comment, marked as answer]
The image is maintained by a [different group of folks](https://github.com/docker-library/rabbitmq).

**Comment by lukebakken** [comment]
@mh21 please start a discussion here, don't just blindly assume this is a bug:

https://github.com/docker-library/rabbitmq/discussions
